modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/Amount.scala

```diff
@@ -0,0 +1,28 @@
+package org.tessellation.rosetta.domain
+
+import org.tessellation.ext.derevo.magnoliaCustomizable.snakeCaseConfiguration
+import org.tessellation.rosetta.domain.currency.Currency
+
+import derevo.circe.magnolia._
+import derevo.derive
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.boolean.Not
+import eu.timepit.refined.generic.Equal
+import io.circe.refined._
+import io.estatico.newtype.macros.newtype
+
+object amount {
+  @derive(customizableDecoder, customizableEncoder)
+  case class Amount(
+    value: AmountValue,
+    currency: Currency
+  )
+
+  type AmountValueRefined = Long Refined Not[Equal[0L]]
+
+  @derive(decoder, encoder)
+  @newtype
+  case class AmountValue(value: AmountValueRefined) {
+    def isNegative: Boolean = value.value < 0
```

Comment:
try this, it should work with no unwrapping:
```scala
def isNegative: Boolean = value < 0L
```

Reply:
eventually try adding `import io.estatico.newtype.ops._` if it won't work

Reply:
By changing `value.value` to `value`, I get `value < is not a member of org.tessellation.rosetta.domain.amount.AmountValueRefine`. I am importing `io.estatico.newtype.ops._`.

Reply:
try with `import eu.timepit.refined.auto._` or `import eu.timepit.refined.cats._`.
it works on my side

Reply:
yes, this is the correct one `import eu.timepit.refined.auto._`

Reply:
Changed. I need help with fixing my test now.